generators/auth/templates/mongodb/jwt/src/utils/pagination.util.ts

```diff
@@ -0,0 +1,82 @@
+import { PaginationParamsInterface } from '@interfaces/pagination-params.interface';
+
+class PaginationUtil {
```

Comment:
why you need a whole class if you can replace him by export const buildLink = () => {}; export const getPaginationLink = () => {} functions ?? I think using the class here is overhead